src/transformers/models/speecht5/tokenization_speecht5.py

```diff
@@ -203,3 +211,11 @@ def save_vocabulary(self, save_directory: str, filename_prefix: Optional[str] =
                 fi.write(content_spiece_model)
 
         return (out_vocab_file,)
+
+    def _normalize(self, text: str):
+        """
+        Normalizes a given string using the 'EnglishNumberNormalizer' class, which converts numeric elements in english
+        text.
+        """
+        normalizer = EnglishNumberNormalizer()
+        return normalizer(text)
```

Comment:
No sure we need a method for this, re-initializing the normalizer on each tokenization.

Reply:
I took the inspiration from Whisper. Instead, should I now initialize the normalizer once in the `__init__()` function and use it directly?